game/src/edit/roads.rs

```diff
@@ -508,8 +508,34 @@ fn can_reverse(_: LaneType) -> bool {
     lt == LaneType::Driving || lt == LaneType::Biking || lt == LaneType::Bus
 }*/
 
+fn default_outside_lane_placement(road: &mut EditRoad, dir: Direction) -> usize {
+    let idx = if road.lanes_ltr.first().unwrap().dir == dir {
+        if road.lanes_ltr.first().unwrap().lt.is_walkable() {
+            1
+        } else {
+            0
+        }
+    } else {
+        if road.lanes_ltr.last().unwrap().lt.is_walkable() {
+            road.lanes_ltr.len() - 1
+        } else {
+            road.lanes_ltr.len()
+        }
+    };
+    idx
+}
+
 // Returns the index where the new lane was inserted
 fn add_new_lane(road: &mut EditRoad, lt: LaneType) -> usize {
+
+    // Default to Fwd as long as the lane directions are equal or there are more with Back
```

Comment:
Style: I got slightly confused reading this, how about `// If there are more lanes pointing forwards, then insert the new one backwards, and vice versa`